src/client.js

```diff
@@ -3020,6 +3020,25 @@ module.exports.CRYPTO_ENABLED = CRYPTO_ENABLED;
  */
 
 /**
+ * Fires whenever an error occurs when call.js sendEvent encounters an error.
```

Comment:
`sendEvent` is an implementation detail. "Fires when the MatrixCall encounters an error when sending a Matrix event".